Makefile

```diff
@@ -189,15 +190,17 @@ e2e:
 fmt:
 	@gofmt -l -w $(SOURCE_DIRS)
 
-$(GOPATH)/bin/golangci-lint:
-	pushd /tmp && GO111MODULE=on go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.39.0 && popd
+golangci-lint:
+	@if $(GOPATH)/bin/golangci-lint version 2>&1 | grep -vq $(GOLANG_CI_LINT_VERSION); then\
```

Comment:
I think it's probably possible to keep using a real path if you version it. Instead of checking `$(GOPATH)/bin/golangci-lint`, the Makefile checks for `$(GOPATH)/bin/golangci-lint-$GOLANGCI_LINT_VERSION`, and we create a `golangci-lint` symlink to the versioned binary. No idea if this is simpler, this is what I was planning to try if nobody fixed that bug first :)